h2o-core/src/main/java/hex/BulkModelBuilder.java

```diff
@@ -0,0 +1,86 @@
+package hex;
+
+import water.H2O;
+import water.Job;
+import water.util.Log;
+
+/**
+ * Runs given model builders in bulk.
+ */
+public class BulkModelBuilder {
```

Comment:
I would rather specialize this class just for CV models building. It still contains references to CV, eg.

     Log.info("Skipping build of last " + (N - i) + " out of " + N + " " + modelType + " CV models");

Moreover, it has certain assumptions about what models are being built. The main assumption is that they will  be somewhat similar in terms of how long it will take to build them. If you have models that take a dramatically different time to build it will become suboptimal in terms of how it is using resources.

Reply:
renamed